docs/usage-as-end-user.md

```diff
@@ -65,6 +65,22 @@ For every zone, the AWS extension creates three subnets:
 For every subnet, you have to specify a CIDR range contained in the VPC CIDR specified above, or the VPC CIDR of your already existing VPC.
 You can freely choose these CIDR and it is your responsibility to properly design the network layout to suit your needs.
 
+You can configure [Gateway VPC Endpoints](https://docs.aws.amazon.com/vpc/latest/userguide/vpce-gateway.html) by adding items in the optional list `networks.vpc.gatewayEndpoints`. Each item in the list is used as a service name and a corresponding endpoint is created for it. All created endpoints point to the service within the cluster's region. For example, consider this (partial) shoot config:
```

Comment:
Could you summarize it and add it here https://github.com/gardener/gardener-extension-provider-aws/pull/17/files#diff-612a4cfc1d7f00d699d1bd3d3465e1f2R57. as a 1 or 2 lines.
Also, you can adapt the existing example https://github.com/gardener/gardener-extension-provider-aws/pull/17/files#diff-612a4cfc1d7f00d699d1bd3d3465e1f2R37-R39 with commented sample values.

Reply:
please review the update